tikv: improve the note for s3-multi-part-size (#23362)
pingcap/docs: tikv-configuration-file.md

> **Note:**
>

> > This configuration was introduced to address backup failures caused by S3 rate limiting. From v6.1.1 onwards, configure this setting with caution. Setting it too large may cause large parts to fail or time out during network jitter.

Reviewer: ```suggestion
> This configuration item was introduced to address backup failures caused by S3 rate limiting. Setting it too large might cause individual part uploads to fail or time out when the network is unstable.
```

> > This configuration item was introduced to address backup failures caused by S3 rate limiting. Starting from TiDB v6.1.1, configure this value with caution. Setting it too large might cause larger upload parts to fail or time out when the network is unstable.

Reviewer: ```suggestion
> This configuration item was introduced to address backup failures caused by S3 rate limiting. Starting from TiDB v6.1.1, configure this value with caution. Setting it too large might cause large upload parts to fail or time out when the network is unstable.
```

> > This configuration was introduced to address backup failures caused by S3 rate limiting. Since this issue has been resolved by [refining the backup data storage structure](/br/br-snapshot-architecture.md#structure-of-backup-files), from v6.1.1 onwards, configure this setting with caution. Setting it too large may cause large parts to fail or time out during network jitter.

Reviewer 2: ```suggestion
> This configuration was introduced to address backup failures caused by S3 rate limiting. From v6.1.1 onwards, configure this setting with caution. Setting it too large may cause large parts to fail or time out during network jitter.
```
